### ISSUE TITLE
Question about fine-tuning of SBERT

### ISSUE THREAD

**Issue by ewayuan** [UKPLab/sentence-transformers#1275]
Hi, I downloaded all-mpnet-base-v2 and train it on my own training dataset with default setting (epoch 1 and learning rate of 2e-5), and then the trained model does improve accuracy. How could I do fine-tuning sbert and find the best parameters?

**Comment by nreimers** [comment]
By training different models and check-Ing which parameters work the best

**Comment by ewayuan** [comment]
> By training different models and check-Ing which parameters work the best

Is there any suggestion for the number of epochs and learning rate tuning?

**Comment by nreimers** [comment]
Number of epochs depends on the training size set. Default learning rate is fine.
